news/2018-05-03-beatmap-spotlights-march-2018.md

```diff
@@ -24,88 +25,88 @@ First of all, congratulations to the winners of the February 2018 Spotlights. Th
 
-An outstanding 1/6 map by the new face [lazyboy007](https://osu.ppy.sh/users/7882675) this month. Despite not having much experience with taiko, he still produced an excellent mapwith consistent structure. Furthermore, the excellent 1/6 usage and control of pattern density extracts the essence of song perfectly, making it a worthy contender for this month's Spotlight!
+An outstanding 1/6 map by the new face [lazyboy007](https://osu.ppy.sh/users/7882675) this month. Despite not having much experience with taiko, he still produced an excellent map with consistent structure. Furthermore, the excellent 1/6 usage and control of pattern density extracts the essence of the song perfectly, making it a worthy contender for this month's Spotlight!
 
 ## osu!catch
 
-[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/nirvana.jpg)](http://osu.ppy.sh/s/694025)
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/nirvana.jpg)](http://osu.ppy.sh/beatmapsets/694025)
 
 Taking the top spot for this edition of the Spotlights is Camellia's rendition of "Nirvana" mapped by [Spectator](https://osu.ppy.sh/users/702598)! While being of lower star rating than the usual Camellia map, it features several patterns that make the map harder than it seems.
 
 There's the sort-of stuttery 1/4 slider patterns, the antiflows, and the hyperchains, all of which don't usually appear in most maps. It also manages to take the cake for its design, featuring clean patterns throughout the entire map, utilizing both smooth curves and sharp angles in different parts which reflect the song.
 
 Spectator never fails to impress both players and mappers with his style throughout all the maps he's created, and that is evident in this map.
 
-[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/element-of-spada.jpg)](http://osu.ppy.sh/s/485733)
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/element-of-spada.jpg)](http://osu.ppy.sh/beatmapsets/485733)
 
-Finally making it's long-overdue arrival in the Ranked section, [Tenshichan](https://osu.ppy.sh/users/1101600)'s "Element of SPADA" is a prime example of the beauty of simplicity. Without attempting any particularly radical patterning or quirky rhythm choices, this approval-length beatmap still manages to interest and challenge the player, continuously shifting and evolving throughout. Perhaps more importantly, it's also an absolute joy to play through.
+Finally making it's long-overdue arrival in the Ranked section, [Tenshichan](https://osu.ppy.sh/users/1101600)'s "Element of SPADA" is a prime example of the beauty of simplicity. Without attempting any particularly radical patterning or quirky rhythm choices, this approval-length beatmap still manages to interest and challenge the player, continuously shifting and evolving throughout. Perhaps more importantly, its also an absolute joy to play through.
```

Comment:
this was right the first time: it's, not its.

Reply:
brain fart oops, thanks for double checking